app/components/op_primer/border_box_table_component.sass

```diff
@@ -15,12 +15,19 @@
     justify-content: flex-start
     align-items: center
 
+    &--row-item
+      &:not(:first-child)
+        margin-left: 6px
+
+      &:not(:last-child)
+        margin-right: 6px
+
```

Comment:
I assume that this was done to avoid that the wrapped text is glued directly to the next column, right? However, this is not ideal. First, it only intends the rows, not the header, thus creatig a misalignment. Second, it relys on the main column being the first one. If that is not the case, it will not work any more. And last but not least, if I do not have a main column at all, it intends unneccessarily. 

<img width="1440" alt="Bildschirmfoto 2024-11-27 um 08 15 09" src="https://github.com/user-attachments/assets/923831dd-e835-44c3-9ca7-45f313f978aa">

I'd rather suggest to give the main column (including its header) some padding.